cmd/ndm_daemonset/app/command/start.go

```diff
@@ -56,13 +54,16 @@ func NewCmdStart() *cobra.Command {
 			filter.Start(filter.RegisteredFilters)
 			// Start starts registering of probes present in RegisteredProbes
 			probe.Start(probe.RegisteredProbes)
-			port := cmd.Flag("port").Value.String()
-			server.ListenPort = port
-			endpointpath := cmd.Flag("metricspath").Value.String()
-			server.MetricsPath = endpointpath
-			metrics.StartingTime = time.Now()
-			// Start HTTP server for /metrics endpoint
-			go server.StartHttpServer()
+			/*
```

Comment:
Why is this commented? Can you add a TODO item here.. if there is some work planned for this piece of code in the future.

Reply:
Yes, we need to support merrics endpoint in http server(for that matter any endpoint would go here). Those things would be considered as new feature and tracked via rally. As far as concern to this piece of code, nothing is pending. Removing this commented code.